module-system/module-implementations/sov-evm/src/evm/tests.rs

```diff
@@ -1,48 +1,25 @@
-use crate::{evm::AccountInfo, Evm};
```

Comment:
What is the reason of it to be unit tests and not integration test?

Reply:
I mean placing test outside of src and testing crate via public interface

Reply:
The core logic of these tests relies on the public `EVM::call` method and could be separated into an integration test. However, to verify the success of the call, we are currently accessing the crate-private  `EVM` state. While we can make the appropriate query method public, it is currently tailored specifically for testing purposes. Instead, I suggest that we postpone changing the test to an integration test until we implement the `RPC` functionality for the `EVM` module. By doing so, we will be able to examine the internal state using the `RPC`, providing a more comprehensive testing approach.

Reply:
Ok, as long as we plan to move to integration test eventually, I am good.